src/ray/ray_exported_symbols.lds

```diff
@@ -35,5 +34,3 @@
 *Java*
 *JNI_*
 *ray*streaming*
-*aligned_free*
-*aligned_malloc*
```

Comment:
Newline?

Reply:
I think we need the new line at the end of the file like most of ray files, it's also similar with ray_version_script.lds also has newline.